src/pages/workspaces/workspace/Workflows.js

```diff
@@ -189,20 +201,60 @@ const FindWorkflowModal = ajaxCaller(class FindWorkflowModal extends Component {
 
     const { synopsis, managers, documentation } = selectedWorkflowDetails || {}
 
+    const errors = validateWorkflowName(editedWorkflowName, this.props.name)
+
     const renderDetails = () => [
       div({ style: { display: 'flex' } }, [
-        div({ style: { flexGrow: 1 } }, [
-          div({ style: { fontSize: 18, fontWeight: 600, margin: '1rem 0 0.5rem' } }, ['Synopsis']),
+        div({ style: { flexGrow: 1, marginTop: '1rem' } }, [
+          !isEditingName ?
+            div([span({ style: styles.sectionTitle }, ['Workflow Name: ']), workflowRename ? `${workflowRename}` : '',
```

Comment:
This starts on one line, and then jumps to a new line on edit. Could we move this below the label to keep everything consistent?